src/component/inputtext/index.tsx

```diff
@@ -56,7 +56,7 @@ function InputText_(
         error: message ? "error" : undefined,
       }}
       input2={{
-        props: { value: props.value },
+        props: { defaultValue: value },
```

Comment:
What does this fix?

Reply:
It fixed warning of providing value without an `onChange` handler

Reply:
I think this messes up the data binding because there is no `onChange` or `onBlur` being passed into this input. I have an alternative approach in https://github.com/venturemark/webclient/pull/483. Let me know what you think.